addon/templates/components/new-view.hbs

```diff
@@ -1 +1,29 @@
-Test!!!
+{{#fbf-modal}}
+  <div class="fbf-modal__header--style-dark">
+      {{#fbf-item}}
+        <div class="fbf-item__primary-icon" {{action 'dismiss'}}>
+          {{#fbf-icon}}close{{/fbf-icon}}
+        </div>
+        <div class="fbf-item__content">
+          {{#fbf-text title=true light=true}}
+            Request a Feature!
+          {{/fbf-text}}
+        </div>
+      {{/fbf-item}}
+  </div>
+  <div class="fbf-modal__content">
+    {{#fbf-text}}
+      To request a feature, you need to describe the feature in a few words. In the description, you can give more details about the requirements the feature must have, or how the feature must work.
```

Comment:
I think it would be nice to mention that in principle something can be considered a feature if it completes the sentence: 'With the FeedbackFruits platform I would like to...'